kokoro/scripts/linux/build.sh

```diff
@@ -31,8 +31,13 @@ if [ $COMPILER = "clang" ]
 then
   CMAKE_C_CXX_COMPILER="-DCMAKE_C_COMPILER=/usr/bin/clang-5.0 -DCMAKE_CXX_COMPILER=/usr/bin/clang++-5.0"
 else
-  # Use newer gcc than default.
-  CMAKE_C_CXX_COMPILER="-DCMAKE_C_COMPILER=/usr/bin/gcc-5 -DCMAKE_CXX_COMPILER=/usr/bin/g++-5"
+  # Specify we want to build with GCC 7 (which supports C++14)
+  sudo add-apt-repository ppa:ubuntu-toolchain-r/test
+  sudo apt-get update
+  sudo apt-get install -y gcc-7 g++-7
+  sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 100 --slave /usr/bin/g++ g++ /usr/bin/g++-7
+  sudo update-alternatives --set gcc "/usr/bin/gcc-7"
+  CMAKE_C_CXX_COMPILER="-DCMAKE_C_COMPILER=/usr/bin/gcc-7 -DCMAKE_CXX_COMPILER=/usr/bin/g++-7"
```

Comment:
Oh, does this mean I can turn swiftshader on for the ubuntu tests now? I couldn't get it to compile before.

Reply:
I'm confused. I only made this change to get SwiftShader building.
Is there another kokoro config I'm not aware of?

Reply:
There is nothing else you need to do. This is the build file for Linux (there is also one for osx and windows). Each kokoro bot calls this file with given flags, the ubuntu bot disables swiftshader as it wouldn't link for me (due to gcc5 I believe). So, I think with this gcc7 change it should work. I'll give it a try on Monday.

Reply:
I only made this change because the GCC Ubuntu builds started failing after rolling forward Swiftshader.
I can only conclude kokoro was already building Swiftshader, on Linux, using gcc5.